package.json

```diff
@@ -11,10 +11,21 @@
     "url": "https://github.com/OlegIlyenko/graphiql-workspace.git"
   },
   "license": "Apache License 2.0",
-  "main": "index.js",
-  "files": ["dist", "index.js", "graphiql-workspace.css"],
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "graphiql-workspace.js",
+    "graphiql-workspace.min.js",
+    "graphiql-workspace.css",
+    "README.md",
+    "CHANGELOG.md"
+  ],
+  "browserify-shim": {
+    "react": "global:React",
+    "react-dom": "global:ReactDOM"
+  },
   "scripts": {
-    "build": "babel src --out-dir dist/",
+    "build": "./build.sh",
```

Comment:
maybe we can still have both definitions: `build`, `quickBuild`. The problem is that uglyfication is very slow, so for quick iteration (during development) it would be better to have a quick option.

I would suggest to move all bash scripts in `scripts` folder and have 2 build scripts as well: `build.sh`, `quickBuild.sh` WDYT?